client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -297,9 +298,36 @@ const Editor = (props: EditorProps): JSX.Element => {
     // TODO this should *probably* be set on focus
     editorRef.current = editor;
     data.editor = editor;
+
+    const storedAccessibilityModes = () => {
+      const accessibilityMode = {
+        isAccessibilityModeOn: props.inAccessibilityMode
+      };
+
+      type AccessibilityMode = typeof accessibilityMode;
+
+      const accessibility = store.get('accessibilityMode') as AccessibilityMode;
+      if (!accessibility) {
+        store.set('accessibilityMode', accessibilityMode);
+      }
+      // Only able to set the arialabel when it is set to true
+      // Otherwise it gets overwritten by the monaco default aria-label
+      if (accessibility?.isAccessibilityModeOn) {
+        editor.updateOptions({
+          ariaLabel:
+            'Accessibility mode set to true press ctrl + e to disable; Press alt F1 for more options'
+        });
+      }
+
+      return accessibility?.isAccessibilityModeOn ?? false;
```

Comment:
```suggestion
      return accessibility;
```
For example, you can do this if `accessibility` is a boolean.